lib/takeaway.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'dish'
+require_relative 'order'
+require 'twilio-ruby'
+require 'dotenv/load'
+
+class Takeaway 
```

Comment:
Again, another impressive approach of clean, effective code that is easy to digest and understand. Your choices for your naming of methods and classes allows someone to walk in and know exactly what you're trying to achieve